server/controllers/api/assessmentDayDataController/index.js

```diff
@@ -73,7 +73,17 @@ const AssessmentDayDataController = {
           await SiteMetadataModel.upsert(
             appDb,
             { study },
-            { $addToSet: { participants: participant } }
+            {
+              $addToSet: {
+                participants: {
+                  Active: 1,
+                  Consent: new Date(),
+                  study,
+                  participant,
+                  synced: new Date(),
+                },
```

Comment:
does it matter that `participants` is no longer a `participant`, but now _contains_ a `participant`?

Thinking of a scenario where something like `participants.name` used to work, but now it would need to be `participants.participant.name`

can/should we spread `participant`?
```suggestion
                participants: {
                  Active: 1,
                  Consent: new Date(),
                  study,
                  synced: new Date(),
                  ...participant,
                },
```

Also just based on `participants` being plural, should this be an array of participant objects?